testing/compatibility/azure/vm.bicep

```diff
@@ -125,7 +126,7 @@ resource virtualMachine 'Microsoft.Compute/virtualMachines@2022-03-01' = {
   location: location
   properties: {
     hardwareProfile: {
-      vmSize: 'Standard_B2als_v2'
+      vmSize: 'Standard_DS1_v2'
```

Comment:
It seems we can actually get the number of CPUs to be increased in azure subscription.